lib/gap_intelligence/models/merchant_pricing_trend.rb

```diff
@@ -0,0 +1,9 @@
+module GapIntelligence
+  class MerchantPricingTrend < Record
+    attributes :values, :headers
+
+    def initialize(attrs = {})
+      @attributes = attrs
+    end
```

Comment:
If we go with original `initialize` then, as you know from documentation, we will end up having 2 instance variables `raw` and `attributes` with same information and since MPT will mostly have tons of data that will never be changed, I think there is not need to keep `raw`. I could change api to return everything inclosed in `attributes` and so that `@attributes = raw.fetch('attributes', raw)` works as expected in original `initialize`, but we will end up having same 2 variables with same huge amount of data in every one of them